### PR TITLE
Add Russian translation

### PR DESCRIPTION
**Pull request by just-boris**
Hello! Lantern is interesting tool and it can be useful for Russian people.

Here is a translation, which helps find it for people who don't know English

### CODE REVIEW

**Comment by just-boris** [comment]
ping.
If you are not interested in this, I am going to remove my repo.

**Comment by xiam** [comment]
Hey @just-boris, sorry for the awfully long response time I was not aware of this PR. I just assigned it to myself, please keep it open. Thanks a lot!

File: getlantern.org/static/locale/lang-ru_RU.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+  "OPEN_INTERNET_FOR_EVERYONE": "Открытый интернет для каждого",
+  "DOWNLOAD_LANTERN": "Скачать Lantern",
+  "DOWNLOAD_LANTERN_MOBILE": "Отправить ссылку для скачивания",
+  "LANTERN_DESCRIPTION_1": "Lantern это бесплатное приложение, которое доставляет быстрый надежный и безопасный доступ к открытому интернету.",
+  "HOW_DOES_LANTERN_WORK": "Как работает Lantern?",
+  "LANTERN_DESCRIPTION_2": "Lantern это интернет-прокси, который дает вам доступ на заблокированные сайты.",
+  "LANTERN_DESCRIPTION_2": "Lantern is an Internet proxy tool that lets you access blocked sites.",
```
Comment (xiam) [inline getlantern.org/static/locale/lang-ru_RU.json]: Hey @just-boris, this entry `LANTERN_DESCRIPTION_2` is duplicated, could you remove it?

**Comment by just-boris** [comment]
@xiam done

**Comment by xiam** [comment]
Thanks for your contribution!

**Pull request merged**